calcom/cal.com
# fix: Broken reschedule link in email created by workflow for team event

## What does this PR do?



Fixes #13869 
Fixed #13885

## Type of change



- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?
add a workflow to event with variable rescheduleLink and create a event check link in the email.

## Checklist
- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
